Dolphin scripts/README.md

```diff
@@ -16,13 +27,26 @@
 6. In Dolphin, under "Scripts", click "Add new Scripts" and select `Scripts/Entrance Randomizer/__main__.py`.
 7. Enjoy and watch logs for errors 🙂
 
+### About the .graphml file
+
+In order to display the generated map take these steps:
+1. Go to https://graphonline.ru/en/
+2. Under "Graph" choose "Import from file" and choose your generated .graphml file
+3. Under "Algorithms" choose "Arrange the graph"
+4. Enjoy your map! You can freely move the dots, and you can use multiple features under the "Algorithms" tab (like "Find all paths" for instance)
+5. You can also save your map at any time by choosing "Graph" --> "Export to file"
+
 ### Known issues and limitations
 
-- To generate a new seed, simply reload the script.
-- Some linked transitions are not spawning at the right entrance and use the default entrance instead. Known cases:
-  - Jungle Canyon from Punchau Shrine
-  - Bittenbinder's Camp from Mysterious Temple
-- One-way transitions are not linked together. This can be a bit confusing when sometimes going back leads to a different area.
+- To generate a new seed, simply reload the script
+- Sometimes a transition does not make you enter the level from the correct entrance, but from the default entrance instead
+- Some seeds will result in impossible to complete configurations, because you might need some items to progress that you don't have yet
+- When using LINKED_TRANSITIONS = False the generated .graphml map will become very hard to read, given the extreme amount of connections that will be drawn
+
+### Solved issues
+
+- All areas will now be guaranteed to be linked to eachother. This means that between any level A and any level B there is always at least 1 route that leads you from one to the other.
+- The game can now be finished because Altar of Ages, St. Claire's Camp, Gates of El Dorado and the Final Bosses all work correctly.
```

Comment:
Not necessary, that comes with PR descriptions and changelogs. And changelogs are shipped with releases since it's still very much a wip towards a first fully functional first version, at which point I'll bother keeping a proper persistant changelog in the repo
```suggestion
```

Reply:
If you say so.
Change added to upcoming commit.